video_recording.go

```diff
@@ -0,0 +1,95 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+	"strings"
+)
+
+type VideoRecordingService struct {
+	client *Client
+}
+
+const videoRecordingsPathPart = "Recordings"
+
+func videoMediaPathPart(recordingSid string) string {
+	return strings.Join([]string{videoRecordingsPathPart, recordingSid, "Media"}, "/")
+}
+
+type VideoRecording struct {
+	Sid             string            `json:"sid"`
+	Duration        uint              `json:"duration"`
+	Status          Status            `json:"status"`
+	DateCreated     TwilioTime        `json:"date_created"`
+	SourceSid       string            `json:"source_sid"`
+	URI             string            `json:"uri"`
+	Size            uint              `json:"size"`
+	Type            string            `json:"type"`
+	ContainerFormat string            `json:"container_format"`
+	Codec           string            `json:"codec"`
+	GroupingSids    map[string]string `json:"grouping_sids"`
+	Links           map[string]string `json:"links"`
+}
+
+type VideoMedia struct {
+	Location string `json:"location"`
+}
+
+type VideoRecordingPage struct {
+	Meta       Meta              `json:"meta"`
+	Recordings []*VideoRecording `json:"recordings"`
+}
+
+type VideoRecordingPageIterator struct {
+	p *PageIterator
+}
+
+// When you make a request to this URL, Twilio will generate a temporary URL for accessing
+// this binary data, and issue an HTTP 302 redirect response to your request. The Recording
+// will be returned in the format as described in the metadata.
+func (vr *VideoRecordingService) Media(ctx context.Context, sid string) (*VideoMedia, error) {
```

Comment:
Would it make sense to have the API resemble this one for Messages Media? https://github.com/kevinburke/twilio-go/blob/master/media.go#L59

Reply:
I saw that but are you sure we need that to download this media? Because when i was looking at this Messages Media it was looking like some complicated stuff (https://www.twilio.com/docs/api/messaging/media#media-list-resource) but in `VideoRecordings` they returning proper link to download it 
```
{
    "location": "https://com.twilio.dev-us1.video.recording.s3.amazonaws.com/RTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
}
```